Question/server/api/AuthApi.js

```diff
@@ -0,0 +1,66 @@
+axios = require('axios');
+const env = require('../../loadEnvironment.js');
+
+const authRootUrl = env.AUTH_URL + '/auth';
+
+const getToken = async (userInfo) => {
+  try {
+    return await axios.post(authRootUrl + '/generate', userInfo, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+  } catch (err) {
+    if (err.code === 'ERR_NETWORK') {
+      throw Object.assign(new Error(err.code), {
+        response: { status: 408 },
+        message: 'Network Error',
+      });
+    }
+    throw err;
+  }
+};
```

Comment:
Question service does not need to getToken, only for User service to log in

Reply:
removed